app/models/category.rb

```diff
@@ -0,0 +1,2 @@
+class Category < ApplicationRecord
+end
```

Comment:
We need validation for presence and uniqueness on name please. Can't have two categories of the same name and we cant have blank categories.

Reply:
added validation check for both presence and uniqueness for category names